closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -994,6 +995,9 @@ private static final class ProfileParser {
         // The java.security properties.
         private final Properties securityProps;
 
+        private LinkedList<String> profileCheckPropertyNames;
+        private LinkedList<String> profileCheckProviderNames;
```

Comment:
Suggest these use `Set<String>` and should be `final`.

Reply:
Done.